core/libdeno/deno.h

```diff
@@ -1,21 +1,27 @@
 // Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
 #ifndef DENO_H_
 #define DENO_H_
+
 #include <stddef.h>
 #include <stdint.h>
+
+#include "buffer.h"
+
 // Neither Rust nor Go support calling directly into C++ functions, therefore
 // the public interface to libdeno is done in C.
 #ifdef __cplusplus
 extern "C" {
 #endif
 
+typedef deno::PinnedBuf::Raw deno_pinned_buf;
```

Comment:
Is this a slice reference? Please add a comment describing the ownership.
It seems PinnedBufPin is the actual buffer - why is it publicly exposed?